mobile/src/main/res/values/strings.xml

```diff
@@ -551,13 +558,14 @@
     <string name="item_update_widget_content_description">Widget for changing Item state</string>
     <string name="item_update_widget_content_description_edit">Edit widget</string>
     <string name="item_update_widget">Widget</string>
-    <!-- %1$s: Item label, %2$s: Action, e.g. "Lights: On" -->
+    <!-- %1$s: Item label, %2$s: Action, e.g. "Lights: On". Use the source string as translation if the punctuation rules are similar to English. -->
```

Comment:
```suggestion
    <!-- %1$s: Item label, %2$s: Action, e.g. "Lights: On". Use the source string as translation if the punctuation rules are the same. -->
```

Reply:
Maybe "Use the source string as translation if the punctuation rules are the same as in English"?

Reply:
Sure, I just wanted to get rid of "similar".